eth/tracers/native/call_flat_test.go

```diff
@@ -0,0 +1,48 @@
+package native_test
```

Comment:
Please add the license

Reply:
good point!
Done.
@s1na will need a re-approval